backend/cmm_helpers.ml

```diff
@@ -256,42 +256,42 @@ let closure_info ~(arity : arity) ~startenv ~is_last =
 
 let alloc_boxedfloat32_header mode dbg =
```

Comment:
For these I would just write `let alloc_boxedfloat32_header (mode : Cmm.Alloc_mode.t) dbg = ...` then you can omit all the module paths, which tend to clutter.